### ISSUE TITLE
fix bool(pda) behavior

### ISSUE THREAD

**Issue by mhmerrill** [mhmerrill/arkouda#229]
add `pdarray.__bool__()` method to reproduce NumPy behavior.

**Comment by mhmerrill** [comment]
looks like we need to put in a
```python
pdarray.__bool__()
```

to emulate NumPy
```python
ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()
```

**Comment by mhmerrill** [comment]
this is addressed by PR #228
